apollo-router/src/plugins/external.rs

```diff
@@ -320,69 +440,425 @@ impl Plugin for ExternalPlugin {
             .instrument(external_service_span())
             .option_layer(request_layer)
             .option_layer(response_layer)
-            .buffered()
+            .buffer(20_000)
```

Comment:
I think it's still better to use `buffered()` until we decide how to proceed on deprecating/removing that.